### PR TITLE
[sailfish-browser] Remove first time usage overlay. Fixes JB#29502

### PR DESCRIPTION
**Pull request by rainemak**
Remove FirstUseOverlay and FullscreenModeHint components and
all references to them.

- When browser launched for the first time, overlay is shown
- First time usage is marked done when chrome is revealed meaning
that chrome is visible, full screen page is shown, or when double
height toolbar bar is active (overlay is at the bottom).

### CODE REVIEW

**Comment by tworaz** [comment]
LGTM

**Comment by rojkov** [comment]
216 deletions! Looks great to me

**Comment by rainemak** [comment]
Cheers :)

Will merge this also to next-next once merge.

File: src/pages/BrowserPage.qml [hunk 1]
```diff
@@ -205,15 +204,16 @@ Page {
     Connections {
         target: WebUtils
         onOpenUrlRequested: {
-            // url is empty when user tapped icon when browser was already open.
+            console.log("onOpenUrlRequested:", url)
```
Comment (rainemak) [inline src/pages/BrowserPage.qml]: Will remove this debug.

**Pull request merged**